_data/sites.json

```diff
@@ -9478,6 +9478,16 @@
             "shopify.com"
         ]
     },
+    
+    {
+        "name": "Shop Your Way",
+        "url": "https://www.shopyourway.com/secured/people/user/000/settings/account",
+        "difficulty": "easy",
+        "notes": "Visit the linked page, then from the left column, click on Account and follow the link to delete your account. Your account will be deleted within 14 days. Note that deleting your account does not cancel your rewards membership.",
```

Comment:
Do you just end up on a page with a button to delete or do you have to fill in forms and such?

Reply:
Yes. The link takes you to your account settings where one of the options given is to delete your account. From there a pop-up is shown confirming the action.

Reply:
Ok, thanks, then we can keep it as `easy`